pkgs/by-name/jb/jbigkit/package.nix

```
@@ -61,26 +61,31 @@ stdenv.mkDerivation (finalAttrs: {
     done
   '';
 
-  installPhase = ''
-    runHook preInstall
+  installPhase =
+    ''
+      runHook preInstall
 
-    install -vDm 644 libjbig/*.h -t "$out/include/"
-    install -vDm 755 pbmtools/{jbgtopbm{,85},pbmtojbg{,85}} -t "$out/bin/"
-    install -vDm 644 pbmtools/*.1* -t "$out/share/man/man1/"
+      install -vDm 644 libjbig/*.h -t "$out/include/"
+      install -vDm 755 pbmtools/{jbgtopbm{,85},pbmtojbg{,85}} -t "$out/bin/"
+      install -vDm 644 pbmtools/*.1* -t "$out/share/man/man1/"
+    ''
+    + lib.optionalString stdenv.hostPlatform.isDarwin ''
+      install -vDm 644 libjbig/libjbig*.a -t "$out/lib/"
+    ''
+    + lib.optionalString (!stdenv.hostPlatform.isDarwin) ''
```

Same remark as @GaetanLepage here

```suggestion
    + lib.optionalString stdenv.hostPlatform.isLinux ''
```